dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -290,6 +292,21 @@ class BGPPeerMgr(Manager):
         vrf, nbr = key.split('|', 1)
         if key not in self.peers:
             cmd = None
+
+            if "local_addr" not in data:
+                syslog.syslog(syslog.LOG_WARNING, 'Peer {}. Error in missing required attribute "local_addr"'.format(key))
+            else:
+                # The bgp session that belongs to a vnet cannot be advertised as the default BGP session.
+                # So we need to check whether this bgp session belongs to a vnet.
+                interface = InterfaceMgr.get_local_interface(self.directory, data["local_addr"])
+                if not interface:
+                    return False
```

Comment:
what should be the log message here?

Reply:
Actually, this is an intermediate state. Maybe it should add some log with INFO level to display this state. Same problem at here https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-fpm-frr/bgpcfgd#L295